apps/common/test/lexical/ast/detection/struct_fields_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Lexical.Ast.Detection.StructFieldsTest do
+  alias Lexical.Ast.Detection
+
+  use Lexical.Test.DetectionCase,
+    for: Detection.StructFields,
+    assertions: [[:struct_fields, :*]],
+    variations: [:match, :function_body, :function_arguments, :module],
+    skip: [
+      [:struct_reference, :*],
+      [:struct_field_value, :*]
+    ]
+
+  test "is true if the cursor is in current module arguments" do
+    assert_detected ~q[%__MODULE__{«»}]
+  end
+
+  test "is true even if the value of a struct key is a tuple" do
+    assert_detected ~q[%User{«favorite_numbers: {3}»}]
+  end
+
+  test "is true even if the cursor is at a nested struct" do
+    assert_detected ~q[%User{«address: %Address{}»]
+  end
+
+  test "is detected if it spans multiple lines" do
+    assert_detected ~q[
```

Comment:
what's the difference between this test and 

https://github.com/lexical-lsp/lexical/blob/89041565265ddbc06c6adb7429a8524ecc7aad6d/apps/common/test/support/lexical/test/detection_case/suite.ex#L118-L122

Reply:
these tests were just copied from what existed before, I didn't audit them to see if they overlapped, though in this example, the distinction is that the struct fields doesn't include the values.